File: Fundamentals-of-Programing141-main/in-class/13_branching/car.py
"""
In-Class Exercise:  Display different messages depending on the age of
                    a car.

File Name: car.py
Course:    CPTR 141
"""

# get the car year from the user
car_year = int(input("Enter car year: "))

print("Your car:")

# for cars built before 1967
if car_year < 1967:
    print("Probably has few safety features.")

# for cars built after 1970
if car_year >1970:
    print("Probably has head rests.")

# for cars built after 1991
if car_year > 1991:
    print("Probably has anti-lock brakes.")

# for cars built after 2001
if car_year > 2001:
    print("Probably has airbags.")

File: Fundamentals-of-Programing142-main/in-class/11_cpp_non-numeric_types/car.cpp
#include <iostream>
using namespace std;

int main(){

    int carYear;
    cout << "Enter you car year: ";
    cin >> carYear;

if (carYear > 1967){
    cout << "Probubly has few safety featurnes."<< endl;
} if (carYear > 1970){
    cout << "pribubly has head rests."<< endl;
} if (carYear > 1991){
    cout << "probably had ABS."<< endl;
} if (carYear > 2001){
    cout << "Probubly has airbags"<< endl;
}

    return 0;
}